lib/streamlit/server/server.py

```diff
@@ -348,7 +345,7 @@ def _create_app(self):
                 AssetsFileHandler,
                 {"path": "%s/" % file_util.get_assets_dir()},
             ),
-            (make_url_path_regex(base, "media/(.*)"), MediaFileHandler),
+            (make_url_path_regex(base, "media/(.*)"), MediaFileHandler, {"path": ""}),
```

Comment:
Do we need this? Not sure what the point of an empty string provides?

Reply:
Yes, I think Tornado sends the options as keyword arguments, so when it's missing the server errors out.

Reply:
Ahh is that because now it's a static file handler?

Reply:
Yes